src/components/navbar/Navbar.css

```diff
@@ -0,0 +1,31 @@
+.navbarContainer {
```

Comment:
Phew, okay it took a bit of digging but I found a solution. Before styling Navbar,  I had to set  `width` in `#root` and `body` to 100% within App.css. Otherwise the page is column like, even after using display: flex and other styling.